resources/views/backend/books/edit.blade.php

```diff
@@ -110,7 +115,7 @@
               </div>
               <div class="form-group">
                 <label for="exampleInputFile">{{ __('books.picture') }}</label>
-                <input type="file" name="picture" id="exampleInputFile" value="{!! old('picture') !!}">
+                <input type="file" name="picture" id="exampleInputFile" value="{{ $book->picture }}">
```

Comment:
Chỗ này thì ng dùng ko biết ảnh đã được upload hay chưa, vì nó ko show ra ảnh.
Mọi ng làm ri nhé:
- 1 thẻ `img` để hiển thị image đã upload => Để họ biết cái hình họ đã upload trông thế nào
- Ẩn cái input file đi
- 1 button `Change`, dùng để đổi ảnh khác
  + Khi họ click nó thì kích hoạt sự kiện click cho input file ẩn trên kia luôn => chọn file
  + Cho hiển thị cái image vừa chọn ở thẻ `img` phía trên (tham khảo: https://stackoverflow.com/questions/4459379/preview-an-image-before-it-is-uploaded)